it/test/uk/gov/hmrc/apiplatformevents/controllers/ApplicationEventsControllerISpec.scala

```diff
@@ -95,7 +96,7 @@ class ApplicationEventsControllerISpec extends ServerBaseISpec with AuditService
          |"oldCallbackUrl": "$oldCallbackUrl",
          |"newCallbackUrl": "$newCallbackUrl"}""".stripMargin
 
-  def validProductionAppNameChangedJsonBody(oldAppName: String, newAppName: String, requestingAdminEmail: LaxEmailAddress): String =
+  def validProductionAppNameChangedJsonBody(oldAppName: ApplicationName, newAppName: ApplicationName, requestingAdminEmail: LaxEmailAddress): String =
```

Comment:
The old name might not be a valid ApplicationName, since it could have been created before the validation was improved.